multicluster/service-mirror/cluster_watcher.go

```diff
@@ -735,7 +735,7 @@ func (rcsw *RemoteClusterServiceWatcher) getMirrorServices() (*corev1.ServiceLis
 	}
 	services, err := rcsw.localAPIClient.Client.CoreV1().Services("").List(context.Background(), metav1.ListOptions{LabelSelector: labels.SelectorFromSet(matchLabels).String()})
 	if err != nil {
-		return nil, err
+		return nil, RetryableError{[]error{err}}
```

Comment:
Though, with the prior recommendation, we probably need to move this wrapper up into the handler so that the outer type is still a RetryableError?

Though, it's not clear to me what errors we think are _not_ retryable or why this type exists to begin with.